NimbleExtension/Sources/JSONMapper/Models/JSONAPIDecoding+LocalizedError.swift

```diff
@@ -0,0 +1,13 @@
+//  JSONAPIErrorSource.swift
+//
+//  Created by Issarapong Poesua on 18/9/18.
+//  Copyright © 2018 Nimbl3. All rights reserved.
+//
+
+enum Errors: Error {
+    enum JSONAPIDecoding: LocalizedError {
```

Comment:
`JSONAPIDecoding` the name is not related to an Error, change the name is better?